docs/_getting_started/modules.md

```diff
@@ -210,20 +232,23 @@ load("rstudio")
 prepend_path("PATH","/software/free/R/R-4.2.1/bin")
 ```
 
-Linux modules only modify your current working environment which means that if you lose connection to the Yens or close your terminal window,
-you will need to reload the modules. But all of the libraries or packages that you have installed as a user will persist and 
-only need to be installed once.
+!!! Important
+    R uses separate libraries for each major version, so you will need to reinstall packages whenever you upgrade your R version. Note that RStudio supports only one R version at a time, and currently, this version is 4.3. If you are an RStudio user, you will need to install packages specifically for this version.
```

Comment:
I think all of the info below is a repeat and folks can just refer to the R page.